corehq/apps/hqwebapp/static/hqwebapp/js/main.js

```diff
@@ -28,6 +29,23 @@ var eventize = function (that) {
     return that;
 };
 
+// This is copy of csrf ajax utilities from hqwebapp/js/ajax_csrf_setup.js
```

Comment:
Cool. I will change it, I did that inspite of thinking about the multiple code update issue